Lint检查失败: rest-channel.ts 应使用数组解构 (prefer-destructuring)

## 测试输出错误信息

```
/Users/hs3180/clawd/disclaude/src/channels/rest-channel.ts
  459:13  error    Use array destructuring       prefer-destructuring
```

## 根本原因分析

在 `src/channels/rest-channel.ts` 第 459 行，代码使用数组索引访问：
```typescript
const chatId = asyncChatMatch[1];
```

ESLint 的 `prefer-destructuring` 规则要求使用数组解构语法：
```typescript
const [, chatId] = asyncChatMatch;
```

## 相关代码上下文

```typescript
    // Async mode: POST /api/chat/{chatId}
    const asyncChatMatch = url.match(new RegExp(`^${this.apiPrefix}/chat/([^/]+)$`));
    if (asyncChatMatch && req.method === 'POST') {
      const chatId = asyncChatMatch[1];  // <-- 第 459 行，应使用解构
      await this.handleAsyncChat(req, res, chatId);
      return;
    }
```

## 触发来源

CI Monitor 定时任务检测到主分支更新后自动运行测试发现：
- Commit: 32fed1b (Merge PR #742 - feat/issue-738-rest-async-mode)
- 修改文件: src/channels/rest-channel.ts (+251 行)

## 修复建议

将第 459 行改为：
```typescript
const [, chatId] = asyncChatMatch;
```

--- a/src/channels/rest-channel.ts
+++ b/src/channels/rest-channel.ts
@@ -456,7 +456,7 @@ export class RestChannel extends BaseChannel<RestChannelConfig> {
     // Async mode: POST /api/chat/{chatId}
     const asyncChatMatch = url.match(new RegExp(`^${this.apiPrefix}/chat/([^/]+)$`));
     if (asyncChatMatch && req.method === 'POST') {
-      const chatId = asyncChatMatch[1];
+      const [, chatId] = asyncChatMatch;
       await this.handleAsyncChat(req, res, chatId);
       return;
     }

